lms/djangoapps/course_home_api/outline/v1/serializers.py

```diff
@@ -66,9 +66,12 @@ class CourseGoalsSerializer(serializers.Serializer):
     """
     Serializer for Course Goal data
     """
-    goal_options = serializers.ListField()
+    goal_options = serializers.SerializerMethodField()
```

Comment:
You did this change to allow `None`? Would `ListField(default=[])` work for same thing? Been a while since I messed with serializers.